src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -136,10 +136,11 @@ function setClipboardMessage(content: string | undefined) {
     if (!Clipboard.canSetHtml()) {
         Clipboard.setString(Parser.htmlToMarkdown(content));
     } else {
-        const anchorRegex = CONST.REGEX_LINK_IN_ANCHOR;
-        const isAnchorTag = anchorRegex.test(content);
-        const plainText = isAnchorTag ? Parser.htmlToMarkdown(content) : Parser.htmlToText(content);
-        Clipboard.setHtml(content, plainText);
+        // For all cases of the "Paste as plain text" action, use only markdown format.
+        // This update ensures consistency across all platforms.
+        // Refer to Issue #53718 for discussion details.
```

Comment:
```suggestion
        // Use markdown format text for the plain text(clipboard type "text/plain") to ensure consistency across all platforms.
        // More info: https://github.com/Expensify/App/issues/53718
```
How about updating the comment?